day7/GroceryStoreInventory.java

```diff
@@ -0,0 +1,66 @@
+package day7;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class GroceryStoreInventory {
+    private static void displayInventory(HashMap<String, Integer> stock) {
+        for(Map.Entry<String, Integer> entry : stock.entrySet()){
+            System.out.println("Name: " + entry.getKey()+","+"Stock Available is "+entry.getValue());
+        }
+    }
+    public static void main(String[] args) {
+        HashMap<String, Integer> stock = new HashMap<>();
+
+
+        stock.put("Bread",20);
+        stock.put("Butter",50);
+        stock.put("Fresh Juice",30);
+        stock.put("Milk",50);
+        stock.put("Eggs",75);
+
+        System.out.println("Current Stock");
+        displayInventory(stock);
+
+        stock.put("Milk",25);
+        stock.put("Eggs",40);
+
+        System.out.println("Updated Stock");
+        displayInventory(stock);
+
+//        String item = "Bananas";
+          String item = "Egg";
```

Comment:
The variable `item` is set to `"Egg"` but the stock contains `"Eggs"`,. As a result your code will print that `"Egg"` is not in stock. To avoid this, you should use the correct key, `"Eggs"`